libnmstate/nm/sriov.py

```diff
@@ -18,6 +18,7 @@
 #
 
```

Comment:
The commit message is missing a justification for this change. Also there should be a bug for NetworkManager to request the missing feature.

Reply:
The date needs to be adjusted to say `2019-2020`.

Reply:
Yes, there is already an existing BZ for NM. I will add it as a reference in the commit message among the justification for this change. Thanks!